endpoints/auction.go

```diff
@@ -400,15 +400,6 @@ func setLabelSource(labels *pbsmetrics.Labels, req *pbs.PBSRequest, status *stri
 	}
 }
 
-func getAdapterValue(bidder *pbs.PBSBidder) openrtb_ext.BidderName {
```

Comment:
Replaced with a simple `openrtb_ext.BidderName(bidder.BidderCode)`, as that's always the end result.